src/bci_build/package/base-fips/README.md.j2

```diff
@@ -2,23 +2,37 @@
 {% include 'badges.j2' %}
 
 ## Description
-This container image is a SUSE Linux Enterprise 15 SP3 base container image
-that includes the SLES 15 FIPS-140-2 certified OpenSSL module.
+This SUSE Linux Enterprise 15 SP3 based container image includes the
+SLES 15 FIPS-140-2 certified OpenSSL and libgcrypt modules.
 
-The FIPS-140-2 certified OpenSSL module is a cryptographic module that provides
-a FIPS-140-2 compliant cryptographic library. The module is certified by the
-National Institute of Standards and Technology (NIST).
+The [FIPS-140-2 certified OpenSSL module](https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3991.pdf)
+is a cryptographic module that provides a FIPS-140-2 compliant
+cryptographic library. The module is certified by the National
+Institute of Standards and Technology (NIST).
 
 The FIPS-140-2 certified OpenSSL module is a drop-in replacement for the
 standard OpenSSL library. It provides the same functionality as the standard
 OpenSSL library, with additional security features to meet the FIPS-140-2
 requirements.
 
+Similarly, the [FIPS-140-2 certified libgcrypt module](https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3848.pdf)
+is a drop-in replacement for the standard libgcrypt library. It provides the
+same functionality as the standard libgcrypt library, with the additional
+security features enforced to meet FIPS-140-2 requirements.
+
 ## Usage
-The image is configured to enforce the use of FIPS-140 mode by default via the
-environment variable `OPENSSL_FORCE_FIPS_MODE`. This variable is set to `1` in
-the image. This means that all cryptographic operations performed by the
-OpenSSL library are done in FIPS-140-2 mode.
+The image is configured to enforce the use of FIPS-140 mode by default independent
+of the host environment setting via the following configured environment variables:
+* `OPENSSL_FIPS=1`: Initialize the OpenSSL FIPS-140 mode
+* `OPENSSL_FORCE_FIPS_MODE=1`: Set FIPS mode to enforcing independent of the host kernel
+* `LIBGCRYPT_FORCE_FIPS_MODE=1`: Set FIPS mode in libgcrypt to enforcing
+
+The following environment variables environment variables are available to configure
+further aspects of the OpenSSL library:
+
+* `OPENSSL_ENFORCE_MODULUS_BITS=1`: Restrict the OpenSSL module to only generate
+the acceptable key sizes of RSA.
```

Comment:
Can you please add a link to further information about this or is this common knowledge in the FIPS world?
```suggestion
* `OPENSSL_ENFORCE_MODULUS_BITS=1`: Restrict the OpenSSL module to only generate
the acceptable key sizes of RSA.
```

Reply:
@dcermak I wish I would have that link